Xamarin.Forms.Platform.iOS/Renderers/SearchBarRenderer.cs

```diff
@@ -120,8 +120,7 @@ protected override void SetBackgroundColor(Color color)
 			
 			Control.BarTintColor = color.ToUIColor(_defaultTintColor);
 
-			if (color.A < 1)
```

Comment:
@rmarinho Do you remember why this `if` statement was here? It came from https://github.com/xamarin/Duplo/pull/3265 and https://github.com/xamarin/Duplo/pull/3304 (fixes for https://bugzilla.xamarin.com/show_bug.cgi?id=31669).

Reply:
Sorry I don't recall tbh.

Reply:
@hartez My take on the logic for the `if` is this: if there's any transparency set on the color, clear the background image so that the tint color actually uses that color with alpha, otherwise it won't work. I think the idea was the clearing the background image is not necessary otherwise, just setting the `BarTintColor` should had worked, but it looks like it doesn't. Not sure if this helps.